au/code/au/utility/test/string_constant_test.cc

```diff
@@ -92,46 +93,46 @@ TEST(IToA, HasLengthMember) {
 }
 
 TEST(UIToA, CanHandleNumbersBiggerThanIntmaxButWithinUintmax) {
-    EXPECT_STREQ(UIToA<10000000000000000000u>::value.c_str(), "10000000000000000000");
```

Comment:
Thanks for adding separators here.  I know it technically makes the "string of source code" not equal to the printed string, but I appreciate making it easier to tell what the actual number is.  (Plus, if the test passes, you _know_ that it's the same number.)